server/utils/licenceStatus.js

```diff
@@ -176,6 +179,7 @@ function getCaStageState(licence) {
       bassAccepted,
       bassWithdrawn,
       bassWithdrawalReason,
+      caRefused: (approvalRelease.decision === 'Yes' && finalChecksRefusal.decision === 'Yes') || false,
```

Comment:
You don't need the `|| false` bit. The value of the  first part of the expression is always either `true` or `false`

Reply:
oops, that was me trying code out but then forgot to take it out.
Done